src/app/service/poap/EndPOAP.ts

```diff
@@ -58,6 +68,7 @@ export default async (guildMember: GuildMember, ctx?: CommandContext): Promise<a
 	const bufferFile = getBufferFromParticipants(listOfParticipants, channel);
 	const currentDate = (new Date()).toISOString();
 	const fileName = `${channel.guild.name}_${channel.name}_${listOfParticipants.length}_participants.csv`;
+	await guildMember.send({ content: 'The event has ended and here are the results.' });
```

Comment:
'You did it! The event has ended\; here are the results:'

Reply:
How about "Finally it's over..." XD